desktop-widgets/modeldelegates.cpp

```diff
@@ -281,6 +282,42 @@ void TankUseDelegate::setModelData(QWidget *editor, QAbstractItemModel *model, c
 	model->setData(index, comboBox->currentIndex());
 }
 
+
+SensorDelegate::SensorDelegate(QObject *parent) : QStyledItemDelegate(parent)
+{
+}
+
+QWidget *SensorDelegate::createEditor(QWidget *parent, const QStyleOptionViewItem &, const QModelIndex &index) const
+{
+	QComboBox *comboBox = new QComboBox(parent);
+	std::vector<int16_t> sensors;
+	const struct divecomputer *currentdc = get_dive_dc(current_dive, dc_number);
+	for (int i = 0; i < currentdc->samples; ++i) {
+		auto &sample = currentdc->sample[i];
+		for (auto s = 0; s < MAX_SENSORS; ++s) {
```

Comment:
consistency question  (I know, idle nit picking)... why an `int` in L295, but an `auto` here?

Reply:
Since this is mostly taken from CylindersModel, I think the inner loop went through a few different iterations in my original sensor patch before I understood the sensor data model.